File: Stella Octangula numbers/stella_octangula_numbers.cpp
#include<bits/stdc++.h>

int main()
{
	int n,t;
	std::cout<<"Stella Octangula numbers\nEnter your choice:\n[1]  Print the single term of the series\n[2] Print series till term specified\n";
	std::cin>>n;
	switch(n)
	{
		case 1:	std::cout<<"[+] Enter the Index:\n";
				std::cin>>t;
				std::cout<<"Term at Index "<<t<<" "<<t*(2*(t*t)-1)<<"\n";
		case 2: std::cout<<"[+] Enter the Index:\n";
				std::cin>>t;
				for(int i=0;i<n+1;++i)
				{
					std::cout<<"Term at Index "<<i<<" "<<i*(2*(i*i)-1)<<"\n";
				}
		default:std::cout<<"Enter Valid option\n";

	}
	return 0;
}

File: Stella Octangula numbers/stella_octangula_numbers.py
#!/usr/bin/env python3

def print_single_term():
	t = int(input('[+] Enter the Index:'))
	print('[*] Term at Index '+str(t))
	print(t*(2*pow(t,2)-1))

def print_series():
	t = int(input('[+] Enter the Index:'))
	for i in range(t+1):
		print('[*] Term at Index '+str(i))
		print(i*(2*pow(i,2)-1))

print('Stella Octangula number series using python')
print('Enter your choice:')
print('[1] Print the single term of the series')
print('[2] Print series till term specified')
n = int(input())
if n == 1:
	print_single_term()
elif n == 2:
	print_series()
else:
	print('[!] Enter valid choice')